pkgs/applications/misc/jiten/default.nix

```
@@ -0,0 +1,79 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, makeWrapper
+, pcre
+, sqlite
+, python3
+, rlwrap
+}:
+python3Packages.buildPythonApplication rec {
+  pname                 = "jiten";
+  version               = "0.4.0";
+  buildInputs           = [ makeWrapper pcre sqlite ];
+  propagatedBuildInputs = with python3Packages; [ click flask python3 rlwrap ];
+  JITEN_VERSION         = version;  # override `git describe`
+  JITEN_ANDROID         = "yes";    # build & package *.sqlite3
+  patchPhase = ''
+    patchShebangs bin/jiten
+    sed -i 's!/bin/bash!'"$(command -v bash)"'!' Makefile jiten/res/jmdict/Makefile
+  '';
+  postFixup = ''
+    wrapProgram $out/bin/jiten --prefix PATH : ${lib.makeBinPath [ python3 rlwrap ]} \
+                               --prefix PYTHONPATH : "$program_PYTHONPATH"
+  '';
```

```suggestion
  propagatedBuildInputs = with python3Packages; [ click flask python3 rlwrap ];

  JITEN_VERSION = version;  # override `git describe`
  JITEN_ANDROID = "yes";    # build & package *.sqlite3

  postPatch = ''
    patchShebangs bin/jiten
    sed -i 's!/bin/bash!'"$(command -v bash)"'!' Makefile jiten/res/jmdict/Makefile
  '';

  postFixup = ''
    wrapProgram $out/bin/jiten \
      --prefix PATH : ${lib.makeBinPath [ python3 rlwrap ]} \
      --prefix PYTHONPATH : "$program_PYTHONPATH"
  '';

```

Can we export the variables in preBuild?